tests/unit_tests/test_sky/server/test_common.py

```diff
@@ -251,37 +251,29 @@ def test_reload_config_for_new_request(mock_all_dependencies, tmp_path,
 
 def test_get_dashboard_url():
     """Test get_dashboard_url with default URL."""
-    common.get_server_url.cache_clear()
     assert common.get_dashboard_url(server_url='http://127.0.0.1:46580'
                                    ) == 'http://127.0.0.1:46580/dashboard'
     """Test get_dashboard_url with basic URL."""
-    common.get_server_url.cache_clear()
     assert common.get_dashboard_url(server_url='http://example.com:8080'
                                    ) == 'http://example.com:8080/dashboard'
     """Test get_dashboard_url with URL containing path."""
-    common.get_server_url.cache_clear()
     assert common.get_dashboard_url(server_url='http://example.com:8080/api/'
                                    ) == 'http://example.com:8080/api/dashboard'
     """Test get_dashboard_url with URL containing credentials."""
-    common.get_server_url.cache_clear()
     assert common.get_dashboard_url(
         server_url='https://user:pass@example.com:8080'
     ) == 'https://example.com:8080/dashboard'
     """Test get_dashboard_url with URL containing username."""
-    common.get_server_url.cache_clear()
     assert common.get_dashboard_url(server_url='https://user@example.com:8080'
                                    ) == 'https://example.com:8080/dashboard'
     """Test get_dashboard_url with host parameter."""
-    common.get_server_url.cache_clear()
     assert common.get_dashboard_url(server_url='http://custom-host:8080'
                                    ) == 'http://custom-host:8080/dashboard'
     """Test get_dashboard_url with complex path."""
-    common.get_server_url.cache_clear()
     assert common.get_dashboard_url(
         server_url='https://user:pass@example.com:8080/api/v1'
     ) == 'https://example.com:8080/api/v1/dashboard'
     """Test get_dashboard_url without port."""
-    common.get_server_url.cache_clear()
```

Comment:
What's the logic with removing these?

Reply:
no longer using an @lru_cache for the `get_server_url()` so this would fail